### ISSUE TITLE
If a new user skips downloading maps, OsmAnd fails to display online tiles by default

### ISSUE THREAD

**Issue by unforgettableid** [osmandapp/OsmAnd#15050]
### How to reproduce?

1. Install OsmAnd.
1. When the new-user UI appears, tap "Get started".
1. When the "Download map" screen appears, tap the "Skip" button, in an attempt to conserve storage space.
1. When the confirmation dialog box appears, tap "Skip" again.

### The problem

1. You'll see a blank screen. No map is visible.

### Explanation

1. The "Online maps" plugin is disabled by default.
1. The "Map source..." setting is set to "Offline vector maps" by default.

### My request

1. On all new installations, please enable the "Online maps" plugin by default.
1. On all new installations, please allow OsmAnd to display online tiles by default, if offline maps are not available or were never downloaded.

Thank you for reading this!

### Your Environment
OsmAnd Version: 4.1.1 from F-Droid
Android/iOS version: Android 8.0.0 "Oreo"
Device model: LG G5

**Comment by scaidermern** [comment]
My suggestion is to not enable online maps automatically. They cause significant data traffic. Instead, _ask_ the user whether to enable online maps if the user chooses to skip downloading offline maps.

**Comment by unforgettableid** [comment]
@scaidermern:

The data traffic is generally no problem over Wi-Fi, but it can indeed be costly when using cellular data.  Especially when the user is using pay-per-megabyte data roaming, which is offered by a few carriers around here.  For example:  Chatr Wireless Canada has traditionally charged about US$0.04/MB for USA roaming, and about US$25/MB for roaming in all other countries.  [(Source.)](https://www.google.com/search?q=Chatr+..+Data+roaming+rates+apply%3A+%240.05%2FMB+U.S.+roaming%2C+%2430%2FMB+international+roaming+(where+available).)

For what it's worth:  Google Maps seems not to worry much about the data traffic involved in fetching online maps.  It generally fetches online maps automatically, without prompting the user.  This only uses maybe ~5 MB per hour of driving.  [(Source.)](https://ting.blog/how-much-data-does-google-maps-use/)  I think Google Maps also uses Wi-Fi to automatically download and cache offline maps, again without prompting the user.

I guess that, if the user is using expensive metered data roaming, Google Maps expects the user to enable its "Wi-Fi only" setting manually.

**Comment by lbdroid** [comment]
I would suspect that most users who have to pay for data roaming would very immediately hit the box that disables data while roaming. If not, I think its a safe assumption for most people that the phone will rack up some roaming data usage, which could cost an unpredictable amount.

**Comment by vshcherb** [comment]
1. It goes a bit against our mission to provide fully functional offline maps
2. It doesn't have most important functions like context menu of points & tracks
3. Search / Navigation / Layers are a bit problematic

**Comment by unforgettableid** [comment]
> 1\. It goes a bit against our mission to provide fully functional offline maps

This is true.  Still, please consider cases in which the user specifically chose to skip the download.  In such cases, it's better to offer them online-only maps than to give them a confusing blank screen with no maps at all.

> 2\. It doesn't have most important functions like context menu of points & tracks
> 3\. Search / Navigation / Layers are a bit problematic

These are good points.  It would probably be good to warn the user about these issues on first run, when they first choose to skip the download of offline maps.  This might encourage them to reconsider their choice.

**Comment by tanc** [comment]
As a new user I'm totally confused about online vs offline maps. I discovered the online maps after a while but I can't seem to plot a route or search for an address. Is routing and searching an offline-only feature? Either way it would be great to be informed about my options at the beginning in some way.

**Comment by vshcherb** [comment]
Yes most of the features offline-only

**Comment by scaidermern** [comment]
> As a new user I'm totally confused about online vs offline maps. I discovered the online maps after a while but I can't seem to plot a route or search for an address. Is routing and searching an offline-only feature? Either way it would be great to be informed about my options at the beginning in some way.

Indeed it would be very nice to have a short list of features on startup that are only usable _with_ offline maps. And also a short explanation for how to use OsmAnd _without_ offline maps.
